etcdctl/ctlv3/command/snapshot_command.go

```diff
@@ -261,18 +263,53 @@ func makeDB(snapdir, dbfile string) {
 	}
 	defer f.Close()
 
+	// get snapshot integrity hash
+	if _, err := f.Seek(-sha256.Size, os.SEEK_END); err != nil {
```

Comment:
probably we want to add a flag called  "no-hash-check"? some people might want to do local snapshot copy. then they do not have a hash attached. and then the integrity checking is on them.